pkg/agent/route/interfaces.go

```diff
@@ -32,7 +32,7 @@ type Interface interface {
 
 	// AddRoutes should add routes to the provided podCIDR.
 	// It should override the routes if they already exist, without error.
-	AddRoutes(podCIDR *net.IPNet, peerNodeName string, peerNodeIP, peerGwIP net.IP) error
+	AddRoutes(podCIDR *net.IPNet, peerNodeName string, peerNodeIP, peerGwIP net.IP, isSvc bool) error
```

Comment:
I'd like to hear @tnqn's opinion as well, but I don't think it makes too much sense to re-use the same function for remote Pod CIDR routes and Cluster IP routes. Just the fact that the name of the first parameter is `podCIDR` is a red flag in my opinion. Why not define a new interface method and do some code unification in the implementation of needed?

Reply:
Agreed, they don't have much similarity in parameters and implementation.

Maybe a new method called `AddServiceRoute`. I don't think it needs peerNodeName, peerNodeIP, peerGwIP

Reply:
Thank you for the suggestion. Use `AddServiceRoutes` now.